scripts/scripts.js

```diff
@@ -231,11 +256,34 @@ async function loadPage() {
     if (lastSection) lastSection.insertAdjacentElement('beforeend', a);
   }
 
-  setConfig({ ...CONFIG, miloLibs: LIBS });
+  let eventConfigItems;
+
+  if (eventUtils) {
+    eventConfigItems = {
+      decorateArea: () => eventUtils?.decorateEvent(document),
```

Comment:
Sorry I missed this one earlier. The decorateArea needs to be declared with a param as such:
```suggestion
      decorateArea: (area = document) => eventUtils?.decorateEvent(area),
```

Reply:
Milo code reference: https://github.com/adobecom/milo/blob/38df7ba5f2f6c17eb67af0097ad2e3510dbda23e/libs/features/personalization/personalization.js#L380